.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+# This configuration file was mainly inspired by ./doc/SETUP_DEVELOPMENT_DOCKER.md
+
+tasks:
+  - name: Rails
+    init: |
+      # install correct ruby+tools versions
+      rvm install 2.6.9 # should be similar as in /Dockerfile-dev 
+      rvm use 2.6.9
```

Comment:
you could perhaps use `rvm install $(cat .ruby-version)` and `rvm use $(cat .ruby-version)`